### DISCUSSION TITLE
Custom Labels for Services and Pods

### DISCUSSION THREAD

**Discussion by jairogutierrezpayu** [arkmq-org/arkmq-org-broker-operator#480] [Q&A] [answered]
Hi, for some policies i need add customized labels to some components generated dinamically such like services, wconsj even control-manager.

Is possible add customized labels to this components

In my case add labels to resources is requirement of FinOps and Security team for all resources in cluster.

Thx

**Comment by brusdev** [comment, marked as answer]
The operator only allow to add customized labels to the the broker pods using the `Spec.DeploymentPlan.Labels` property.
You can customize the control-manager labels editing the `activemq-artemis-controller-manager` deployment, see https://github.com/artemiscloud/activemq-artemis-operator/blob/main/deploy/operator.yaml

It should be easy to extend the `Spec.DeploymentPlan.Labels` to the other resources. Could you provide a use case that requires this feature?

**Reply by jairogutierrezpayu** [reply]
In my case add labels to resources is requirement of FinOps and Security team for all resources in cluster.

**Reply by brusdev** [reply]
I created the issue #481 from this discussions.
